### PR TITLE
Implemented endpoint to fetch random torrents

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by devos50** [comment]
@lfdversluis can you review this one?

File: Tribler/Core/Modules/restapi/torrents_endpoint.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                        "num_seeders": 42,
+                        "num_leechers": 184,
+                        "last_tracker_check": 1463176959
+                    }]
+                }
+        """
+        limit_torrents = 10
+
+        if 'limit' in request.args and len(request.args['limit']) > 0:
+            limit_torrents = request.args['limit'][0]
```
Comment (lfdversluis) [inline Tribler/Core/Modules/restapi/torrents_endpoint.py]: We don't want to return an error in case limit <= 0?

**Comment by lfdversluis** [comment]
Comments given

**Comment by devos50** [comment]
@whirm can you review?

**Pull request merged**